### ISSUE TITLE
installing from shrinkwrap can't find packages

### ISSUE THREAD

**Issue by stesla** [DeadAlready/npm-ducttape#4]
I am using npm 3.10.8 and npm-ducttape 1.1.0.

I run `npm-ducttape vendor/node_packages --dev` and in the npm-shrinkwrap.json there are many stanzas like this:

``` json
    "accepts": {
      "dependencies": {
        "mime-db": {
          "dependencies": {},
          "version": "1.12.0",
          "from": "mime-db@>=1.12.0 <1.13.0",
          "resolved": "file:../../vendor/node_packages/mime-db-1.12.0.tgz",
          "dev": true
        },
        "mime-types": {
          "dependencies": {},
          "version": "2.0.14",
          "from": "mime-types@>=2.0.4 <2.1.0",
          "resolved": "file:../../vendor/node_packages/mime-types-2.0.14.tgz",
          "dev": true
        }
      },
      "version": "1.1.4",
      "from": "accepts@1.1.4",
      "resolved": "file:vendor/node_packages/accepts-1.1.4.tgz",
      "dev": true
    }
```

You'll note that the nested dependencies have "../../" in front of their path. When I then go to run `npm install` in my project directory (`/home/stesla/myproject`) I get errors like this:

```
npm ERR! addLocal Could not install /home/stesla/vendor/node_packages/mime-db-1.12.0.tgz
npm ERR! addLocal Could not install /home/stesla/vendor/node_packages/mime-types-2.0.14.tgz
```

I can get the install to work just fine, if I simply remove all of the "../" components from all the files:

```
 sed -i.old -e 's;\.\./;;g' npm-shrinkwrap.json 
```

**Comment by stesla** [comment]
Notably, on npm 3.10.3, the original npm-shrinkwrap.json with the relative paths works as expected.

**Comment by DeadAlready** [comment]
The relative paths were necessary for the install to succeed as with previous NPM versions it used the dependency folder for install start path when dealing with nested dependencies. However, the newest version of NPM has indeed changed this behaviour...
I'll try to deploy a fix today.

**Comment by DeadAlready** [comment]
I've released v1.2.0 - this will consider the NPM version when creating the shrinkwrap file paths.
However it is important to note, that if the shrinkwrap file was created with a different version of NPM than the install, then the install will still fail.

Here I am talking about 3.10.8 and newer vs older
